smoke_test/tests/unauthenticated/sam_version_tests.py

```diff
@@ -0,0 +1,18 @@
+import json
+
+from ..sam_smoke_test_case import SamSmokeTestCase
+
+
+class SamVersionTests(SamSmokeTestCase):
```

Comment:
what do you see as the benefit in checking the version endpoint?

Reply:
It's just something we can hit without authenticating.  Doesn't add a _ton_ of value, but also doesn't cost much either.  Happy to remove it if we think it's silly.